app/config/variables.php

```diff
@@ -386,7 +386,7 @@
                 'question' => '',
             ],
             [
-                'name' => '_APP_FUNCTIONS_ENVS',
+                'name' => '_APP_FUNCTIONS_RUNTIMES',
```

Comment:
Maybe we should add a new key, and mention the old one is deprecated on 0.8 (and push it to the bottom)

Reply:
Added 👍🏻